Does not work with remote execution

Here is an error that appears when I try a remote execution build:

```
external/com_github_bazelbuild_buildtools/build/BUILD.bazel:5:1: Couldn't build file external/com_github_bazelbuild_buildtools/build/parse.y.baz.go: Executing genrule @com_github_bazelbuild_buildtools//build:parse.y.go_yacc failed: No usable spawn strategy found for spawn with mnemonic Genrule. Are your --spawn_strategy or --strategy flags too strict?
```

Here are my bazelrc flags:
```
build:remote --spawn_strategy=remote
build:remote --strategy=Javac=remote
build:remote --strategy=Closure=remote
build:remote --strategy=Genrule=remote
build:remote --define=EXECUTOR=remote
```

Here's how I install buildtools:
```
load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")

http_archive(
    name = "com_github_bazelbuild_buildtools",
    sha256 = "96def97196a4d8182de97dccf49afc4097a37d451bc9219b7052e8669421f967",
    strip_prefix = "buildtools-0.22.0",
    url = "https://github.com/bazelbuild/buildtools/archive/0.22.0.zip",
)
```

Any ideas what causes it?

[Repository](https://gist.github.com/Monnoroch/86358c3ec7bf81c87d6a84696d90ac19).

Build log (RBE on GCP):
```
starting build "0c0fa82d-b95e-476b-a910-16e1451e7db5"

FETCHSOURCE
Initialized empty Git repository in /workspace/.git/
From https://source.developers.google.com/p/test-astra/r/test-astra
* branch 7120f64a56a4c73015e80750d012c46ffe4e4403 -> FETCH_HEAD
HEAD is now at 7120f64 f
BUILD
Already have image (with digest): gcr.io/cloud-builders/bazel
Extracting Bazel installation...
Starting local Bazel server and connecting to it...
INFO: Invocation ID: 70f84bce-1c15-4c49-a72f-643c8d50933c
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Analyzing: 4 targets (1 packages loaded, 0 targets configured)
Analyzing: 4 targets (1 packages loaded, 0 targets configured)
Analyzing: 4 targets (1 packages loaded, 0 targets configured)
Analyzing: 4 targets (7 packages loaded, 13 targets configured)
Analyzing: 4 targets (11 packages loaded, 27 targets configured)
Analyzing: 4 targets (11 packages loaded, 27 targets configured)
Analyzing: 4 targets (11 packages loaded, 27 targets configured)
Analyzing: 4 targets (11 packages loaded, 27 targets configured)
Analyzing: 4 targets (24 packages loaded, 544 targets configured)
Analyzing: 4 targets (42 packages loaded, 713 targets configured)
Analyzing: 4 targets (50 packages loaded, 733 targets configured)
Analyzing: 4 targets (50 packages loaded, 733 targets configured)
DEBUG: Rule 'com_google_protobuf' modified arguments {"shallow_since": "1551387314 -0800"}
Analyzing: 4 targets (60 packages loaded, 1217 targets configured)
Analyzing: 4 targets (60 packages loaded, 1217 targets configured)
Analyzing: 4 targets (60 packages loaded, 1217 targets configured)
WARNING: /workspace/BUILD.bazel:13:12: in deps attribute of sh_test rule //:test_test: genrule rule '//:test_rule' is unexpected here (expected sh_library); continuing anyway
INFO: Analysed 4 targets (67 packages loaded, 7333 targets configured).
INFO: Found 2 targets and 2 test targets...
[0 / 1] checking cached actions
INFO: From Compiling external/com_google_protobuf/src/google/protobuf/compiler/cpp/cpp_message.cc [for host]:
external/com_google_protobuf/src/google/protobuf/compiler/cpp/cpp_message.cc:525:16: warning: private field 'cold_threshold_' is not used [-Wunused-private-field]
const double cold_threshold_;
^
1 warning generated.
[130 / 257] 1 / 2 tests; Compiling external/com_google_protobuf/src/google/protobuf/compiler/java/java_enum.cc [for host]; 6s remote ... (50 actions running)
[144 / 257] 1 / 2 tests; Compiling external/com_google_protobuf/src/google/protobuf/compiler/java/java_enum.cc [for host]; 15s remote ... (50 actions running)
[179 / 326] 1 / 2 tests; Compiling external/com_google_protobuf/src/google/protobuf/source_context.pb.cc [for host]; 26s remote ... (37 actions running)
[193 / 326] 1 / 2 tests; Compiling external/com_google_protobuf/src/google/protobuf/compiler/cpp/cpp_enum_field.cc [for host]; 37s remote ... (23 actions running)
INFO: From Compiling external/com_google_protobuf/src/google/protobuf/util/json_util.cc [for host]:
external/com_google_protobuf/src/google/protobuf/util/json_util.cc:132:16: warning: 'InvalidName' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
virtual void InvalidName(const converter::LocationTrackerInterface& loc,
^
external/com_google_protobuf/src/google/protobuf/util/internal/error_listener.h:58:16: note: overridden virtual function is here
virtual void InvalidName(const LocationTrackerInterface& loc,
^
external/com_google_protobuf/src/google/protobuf/util/json_util.cc:144:16: warning: 'InvalidValue' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
virtual void InvalidValue(const converter::LocationTrackerInterface& loc,
^
external/com_google_protobuf/src/google/protobuf/util/internal/error_listener.h:63:16: note: overridden virtual function is here
virtual void InvalidValue(const LocationTrackerInterface& loc,
^
external/com_google_protobuf/src/google/protobuf/util/json_util.cc:153:16: warning: 'MissingField' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
virtual void MissingField(const converter::LocationTrackerInterface& loc,
^
external/com_google_protobuf/src/google/protobuf/util/internal/error_listener.h:68:16: note: overridden virtual function is here
virtual void MissingField(const LocationTrackerInterface& loc,
^
3 warnings generated.
[212 / 326] 1 / 2 tests; Compiling external/com_google_protobuf/src/google/protobuf/util/internal/protostream_objectwriter.cc [for host]; 40s remote ... (5 actions, 4 running)
INFO: From Generating Descriptor Set proto_library @com_google_protobuf//:field_mask_proto:
external/com_google_protobuf: warning: directory does not exist.
INFO: From Generating Descriptor Set proto_library @com_google_protobuf//:wrappers_proto [for host]:
external/com_google_protobuf: warning: directory does not exist.
INFO: From Generating Descriptor Set proto_library @com_github_bazelbuild_buildtools//build_proto:blaze_query_proto:
external/com_github_bazelbuild_buildtools: warning: directory does not exist.
INFO: From Generating Descriptor Set proto_library @com_google_protobuf//:empty_proto [for host]:
external/com_google_protobuf: warning: directory does not exist.
INFO: From Generating Descriptor Set proto_library @com_google_protobuf//:struct_proto:
external/com_google_protobuf: warning: directory does not exist.
INFO: From Generating Descriptor Set proto_library @com_google_protobuf//:any_proto [for host]:
external/com_google_protobuf: warning: directory does not exist.
INFO: From Generating Descriptor Set proto_library @com_google_protobuf//:any_proto:
external/com_google_protobuf: warning: directory does not exist.
INFO: From Generating Descriptor Set proto_library @com_google_protobuf//:field_mask_proto [for host]:
external/com_google_protobuf: warning: directory does not exist.
INFO: From Generating Descriptor Set proto_library @com_google_protobuf//:struct_proto [for host]:
external/com_google_protobuf: warning: directory does not exist.
INFO: From Generating Descriptor Set proto_library @com_google_protobuf//:empty_proto:
external/com_google_protobuf: warning: directory does not exist.
INFO: From Generating Descriptor Set proto_library @com_google_protobuf//:source_context_proto:
external/com_google_protobuf: warning: directory does not exist.
INFO: From Generating Descriptor Set proto_library @com_google_protobuf//:timestamp_proto [for host]:
external/com_google_protobuf: warning: directory does not exist.
INFO: From Generating Descriptor Set proto_library @com_google_protobuf//:timestamp_proto:
external/com_google_protobuf: warning: directory does not exist.
INFO: From Generating Descriptor Set proto_library @com_google_protobuf//:source_context_proto [for host]:
external/com_google_protobuf: warning: directory does not exist.
INFO: From Generating Descriptor Set proto_library @com_google_protobuf//:descriptor_proto:
external/com_google_protobuf: warning: directory does not exist.
INFO: From Generating Descriptor Set proto_library @com_google_protobuf//:duration_proto:
external/com_google_protobuf: warning: directory does not exist.
INFO: From Generating Descriptor Set proto_library @com_github_bazelbuild_buildtools//build_proto:blaze_query_proto [for host]:
external/com_github_bazelbuild_buildtools: warning: directory does not exist.
INFO: From Generating Descriptor Set proto_library @com_google_protobuf//:duration_proto [for host]:
external/com_google_protobuf: warning: directory does not exist.
INFO: From Generating Descriptor Set proto_library @com_google_protobuf//:descriptor_proto [for host]:
external/com_google_protobuf: warning: directory does not exist.
INFO: From Generating Descriptor Set proto_library @com_google_protobuf//:wrappers_proto:
external/com_google_protobuf: warning: directory does not exist.
INFO: From Generating Descriptor Set proto_library @com_google_protobuf//:type_proto:
external/com_google_protobuf: warning: directory does not exist.
INFO: From Generating Descriptor Set proto_library @com_google_protobuf//:type_proto [for host]:
external/com_google_protobuf: warning: directory does not exist.
INFO: From Generating Descriptor Set proto_library @com_google_protobuf//:compiler_plugin_proto:
external/com_google_protobuf: warning: directory does not exist.
INFO: From Generating Descriptor Set proto_library @com_google_protobuf//:compiler_plugin_proto [for host]:
external/com_google_protobuf: warning: directory does not exist.
INFO: From Generating Descriptor Set proto_library @com_google_protobuf//:api_proto:
external/com_google_protobuf: warning: directory does not exist.
INFO: From Generating Descriptor Set proto_library @com_google_protobuf//:api_proto [for host]:
external/com_google_protobuf: warning: directory does not exist.
[258 / 326] 1 / 2 tests; Generating into bazel-out/host/bin/external/io_bazel_rules_go/proto/wkt/linux_amd64_stripped/any_go_proto%/github.com/golang/protobuf/ptypes/any [for host]; 2s remote ... (25 actions, 24 running)
[307 / 326] 1 / 2 tests; GoCompile external/com_github_bazelbuild_buildtools/api_proto/linux_amd64_stripped/go_default_library%/github.com/bazelbuild/buildtools/api_proto.a [for host]; 2s remote ... (4 actions, 3 running)
FAIL: //:lint-buildifier (see /builder/home/.cache/bazel/_bazel_root/eab0d61a99b6696edb3d2aff87b585e8/execroot/test_astra/bazel-out/k8-fastbuild/testlogs/lint-buildifier/test.log)
INFO: From Testing //:lint-buildifier:
==================== Test output for //:lint-buildifier:
./BUILD.bazel # reformat 
./WORKSPACE # reformat callsort
================================================================================
INFO: Elapsed time: 178.572s, Critical Path: 96.51s
INFO: 312 processes: 136 remote cache hit, 2 local, 174 remote.
INFO: Build completed, 1 test FAILED, 327 total actions
//:test_test (cached) PASSED in 1.6s
//:lint-buildifier FAILED in 2.2s
/builder/home/.cache/bazel/_bazel_root/eab0d61a99b6696edb3d2aff87b585e8/execroot/test_astra/bazel-out/k8-fastbuild/testlogs/lint-buildifier/test.log

Executed 1 out of 2 tests: 1 test passes and 1 fails remotely.
INFO: Build completed, 1 test FAILED, 327 total actions
ERROR
ERROR: build step 0 "gcr.io/cloud-builders/bazel" failed: exit status 3
```

--- a/build/build_defs.bzl
+++ b/build/build_defs.bzl
@@ -33,7 +33,6 @@ def go_yacc(src, out, visibility = None):
                " $(location " + _GO_YACC_TOOL + ") " +
                " -o $(location " + out + ") $(SRCS) > /dev/null"),
         visibility = visibility,
-        local = 1,
     )
 
 def _extract_go_src(ctx):

